### ISSUE TITLE
260 publication, distribution, etc. (imprint)

### ISSUE THREAD

**Issue by CECSpecialistI** [uwlib-cams/MARC2RDA#125]
https://github.com/uwlib-cams/MARC2RDA/blob/main/Working%20Documents/2XX.csv

**Comment by CECSpecialistI** [comment]
@szapoun discussion in issue #129 could pertain to this mapping as well. Be sure to avoid soft-deprecated properties in favor of properties in alignment at [RDA Registry](https://www.rdaregistry.info/Aligns/alignSoft2Rec.html)

**Comment by szapoun** [comment]
@CECSpecialistI I have updated the spreadsheet for both 260 and 264. Added in the comments the change. I also changed status to first pass.

**Comment by szapoun** [comment]
@gerontakos I have updated the status

**Comment by pennylenger** [comment]
@szapoun Hi Sofia, for 260, subfields are mapped to Place of publication, Name of publisher, Date of publication or publication statement in first pass by you. I found on MARC21 that it is information relating to the publication, printing, distribution, issue, release, or production of a work. So is there any specific reason why they are only mapped to publication related elements? And the first indicator is all about publisher, but subfield a, b, c also mentions distribution, etc. I have changed the mapping to place, name and date of manifestation. If there is a reason for them to be mapped to publication related elements, I will change them.

**Comment by AdamSchiff** [comment]
Field 260 was used for publication, distribution, and manufacture information.  Publisher and distributor information was not distinguished by the subfields $a, $b, and $c, but by some indication that the data recorded was related to distribution (otherwise it was assumed to be publication information).  $e, $f, and $g were used for manufacture information.

A person might be able to tell when distribution info was recorded, because it could be preceded with a phrase like "Distributed by" or "For sale by" or followed by a bracketed term like "[distributor]".  To identify which data is pertaining to distribution, we'd have to come up with a list of these phrases.  There are a few common ones, but I'm sure we'd miss many others.

Adam

Adam L. Schiff
Principal Cataloger
University of Washington Libraries
(206) 543-8409
***@***.***
________________________________

**Comment by szapoun** [comment]
@pennylenger Thank you Penny for correcting. You are right. 

Regarding distribution, do you think we can add a rule that if " distribut* " exists in 260$b, then 260$b should be mapped as "rdam:P30173 "has name of distributor"". Not sure about $a and $c in this case? @AdamSchiff what do you think?

**Comment by lake44me** [comment]
@CECSpecialistI @pennylenger Please note - before this mapping is ready for transform, the rows that get [Rreproduction Conditions](https://github.com/uwlib-cams/MARC2RDA/wiki/Reproductions-Guidance) checked need to have that information added. Let @lake44me and @dchen077 know when the mappings/transform are otherwise reviewed/approved.

**Comment by cspayne** [comment]
@pennylenger @szapoun It looks like all first pass rows have been reviewed, is this field ready for transform?

**Comment by pennylenger** [comment]
Yes, it's ready.

**Comment by cspayne** [comment]
@AdamSchiff Will '=' ever appear in 260 $a? I can't find any indication that this is used and I can't find any examples in MARC records. I've only found '=' in $b.

**Comment by cspayne** [comment]
Hi @szapoun, I noticed that the mapping of a, b, and c together as publication statement was deleted. We have this mapped in 264. I think a, b, and c should be mapped to publication statement unless $b contains distribut* or "for sale by" in which case it should map to distribution statement. What do you think?

**Comment by tmqdeborah** [comment]
I definitely agree that we should map the 260 data, and suggest that we might be able to do so using terms as conditions such as the below. There may be other terms that could be added for each of the categories below.

Name of distributor, if
- single or subsequent $b contains:
  - 'dist*' or 'sold' or 'marketed' or 'sale by' or 'exported' or 'imported' or 'offered' or 'supplied' or 'obtained from' or 'circulated' or 'available' or 'copies my be obtained' or 'leased by' or 'offered by'

Name of manufacturer, if
- subsequent $b contains:
  - 'print*' or 'manufacture*' or 'bound' or 'binder' 

Name of producer, if
- single 'produced' or 'photographed by' or 'photo* by' or 'engr.' or 'engraved' or 'filmed by' or 'lith. by' or 'lithogr. by' or 'lithographed by' or 'made by'

Name of publisher, if
- single $b that contains reproduction terms: 
  - 'reproduced' or 'mimeographed' or 'duplicated' or 'facsim*' or 'microfilmed by' or 'stereotyped by'
- single $b that contains:
  - 'print*' or 'manufacture*' or 'bound' or 'binder' or 'imprynted'"
- single or subsequent $b that does not contain any Distribution, Manufacture, or Producer terms (i.e., contains no terms or contains any terms other than those terms, e.g., 'prepared by', 'developed by' or 'presented by' or 'released by' or 'administered' or 'assisted by' )

Terms are not case-sensitive
Base choice for 'place of ...' and 'date of ...' on choice from $b above

**Comment by cspayne** [comment]
Yes! This 260 data is getting mapped. More specifically, I was asking about additionally mapping the subfields together as "has publication statement" and "has distribution statement", alongside the individual mappings of each subfield as "name of..." and "place of..." properties. This is how we did 264, and it looks like we did map it this way at one point but the row with this mapping was marked delete.

**Comment by tmqdeborah** [comment]
Good to hear the 260 will be mapped; I saw that the 264 was mapping in some of the records in the most recent review file, but the 260 wasn't. I should have checked the status in the spreadsheet, sorry.

I don't know when/why it was deleted, but I think it would be even more useful to add statements from 260 than it is to add them from 264 for has "publication statement" and "has distribution statement", etc.

**Comment by szapoun** [comment]
Hi Cypress! Let's be consistent anc copy the 264 mapping to the 260 field also. Thank you

**Comment by cspayne** [comment]
> Name of manufacturer, if
>
>    subsequent $b contains:
>        'print*' or 'manufacture*' or 'bound' or 'binder'

@tmqdeborah won't name of manufacturer be in $f for 260, not $a?

**Comment by tmqdeborah** [comment]
> > Name of manufacturer, if
> > subsequent $b contains:
> > 'print*' or 'manufacture*' or 'bound' or 'binder'
> 
> [@tmqdeborah](https://github.com/tmqdeborah) won't name of manufacturer be in $f for 260, not $a?

Absolutely! Good catch!

**Comment by cspayne** [comment]
This may need further string processing/clean up as real record output gets reviewed, but the base coding is complete.
